stdlib/public/CTensorFlow/CMakeLists.txt

```diff
@@ -98,17 +98,6 @@ foreach(sdk ${TARGET_SDKS})
       COMPONENT stdlib
       )
   endforeach()
-
-  # Install TensorFlow libraries in each SDK subdirectory.
```

Comment:
Note: this installation logic is in conflict with the `Installing tensorflow` logic in `utils/build-script-impl` and caused issues in practice, overwriting `swift-nightly-install/usr/lib/swift/<platform>/libtensorflow.swift`. It can be removed.